src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -130,6 +131,13 @@ def post(self, request, project):
             )
             report = existing_report
 
+        if report.group_id is None:
+            backfill_group.apply_async(
```

Comment:
We should also explicitly delay this here, like, 30 seconds or so just to give it time. Otherwise, we're literally running the background job at the exact same time we just checked.

Reply:
good point

Reply:
done